src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -239,6 +249,10 @@ public String locationWithLangCode(String location) {
         }
         path = pathNormalize(path);
 
+        if (!path.startsWith(this.settings.sitePrefixPathWithoutSlash)) {
```

Comment:
Will this still behave correctly when they aren't using "site prefix path"?

Reply:
It seems to work well.

`sitePrefixPathWithoutSlash = ""` by default.
`path.startsWith("")` will always be true.